client/cmddata.c

```diff
@@ -919,7 +919,9 @@ int CmdDetectClockRate(const char *Cmd)
 
 char *GetFSKType(uint8_t fchigh, uint8_t fclow, uint8_t invert)
 {
-	char *fskType;
+	static char fType[8];
+	memset(fType, 0x00, 8);
+	char *fskType = fType;
 	if (fchigh==10 && fclow==8){
 		if (invert) //fsk2a
 			fskType = "FSK2a";
```

Comment:
But I think this is problematic. We're now reassigning fsktype to a compile time string. I'm not sure about that... It could be fine, I guess, but what happens to the static area of memory we allocated earlier? I'd feel more comfortable if it was done via a memcpy into the `fType` area.